src/Module/Hashtag.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @file src/Module/Hashtag.php
+ */
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Core\System;
+use dba;
+
+/**
+ * Hashtag module.
+ */
+class Hashtag extends BaseModule {
+
+	public static function init()
```

Comment:
This works of course, but I'd rather use the `content()` function instead. `init()` is akin to a constructor, with a module initialization code that should be ran whether the request is GET or POST. If we only expect GET, then we should just use `content()`.